shared/utils/verify-input-sanity.py

```diff
@@ -31,7 +31,7 @@
        <generator>
         <oval:product_name>testcheck.py</oval:product_name>
         <oval:product_version>0.0.1</oval:product_version>
-        <oval:schema_version>5.10</oval:schema_version>
+        <oval:schema_version>5.11</oval:schema_version>
```

Comment:
I would say it matters. The ```verify-input-sanity.py``` checking script should verify particular OVAL check sanity for syntactic correctness. Obviously due to changes in OVAL language 5.10.1 vs 5.11 vs 5.11.1 some elements changed in the sense they aren't inter-version compatible. So would be better RHEL checks to be checked explicitly against 5.10.1 version and introduce own ```verify-input-sanity.py``` copy for Fedora content.